features/step_definitions/ui/view-and-card-brand-products.steps.js

```diff
@@ -0,0 +1,29 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+const { expect } = require("@playwright/test");
+
+Then("I verify that Brands are visible on left side bar", async function () {
+  const brandsSidebar = await this.page.locator('.brands_products');
+  await expect(brandsSidebar).toBeVisible();
+});
+
+When("I click on MADAME brand link", async function () {
+  const brandLink = await this.page.locator('a[href="/brand_products/Madame"]');
+  await brandLink.click();
+});
+
+Then("I verify that user is navigated to brand page and brand products are displayed", async function () {
+  const brandTitle = await this.page.locator('h2.title.text-center');
+  await expect(brandTitle).toContainText('Brand - Madame Products');
```

Comment:
do not hard code values get them from the step.

You can define like this.
"I verify that user is navigated to {string}  brand page and brand products are displayed"